tdm-admin/src/app/page/Database.tsx

```diff
@@ -0,0 +1,151 @@
+import IDCell from '~/app/components/database/IDCell';
+import StatusCell from '~/app/components/database/StatusCell';
+import TextCell from '~/app/components/database/TextCell';
+import { database } from '~/app/services/admin/database';
+
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import Pagination from '@mui/material/Pagination';
+import Paper from '@mui/material/Paper';
+import Skeleton from '@mui/material/Skeleton';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo, useState } from 'react';
+
+import type { ChangeEvent } from 'react';
+
+const DatabaseSkeleton = () => {
+    return (
+        <Paper>
+            <Box sx={{ padding: '12px' }}>
+                <Skeleton component="div" sx={{ width: '100%' }} animation="wave" variant="rounded" height={600} />
+                <Skeleton
+                    sx={{
+                        marginLeft: 'auto',
+                        marginTop: '12px',
+                        position: 'relative',
+                    }}
+                    animation="wave"
+                    variant="rounded"
+                    height={32}
+                    width={230}
+                />
+            </Box>
+        </Paper>
+    );
+};
+
+const Database = () => {
+    const [page, setPage] = useState(1);
+
+    const { data, isFetching, isLoading } = useQuery({
+        queryKey: ['database', page],
+        queryFn: () => {
+            return database(page);
+        },
+    });
+
+    const numberOfPage = useMemo(() => {
+        if (data) {
+            return Math.round(data.total / 10);
+        }
+        return 0;
+    }, [data]);
+
+    const handlePageChange = (_: ChangeEvent<unknown>, newPage: number) => {
+        setPage(newPage);
+    };
+
+    if (isFetching || isFetching) {
+        return <DatabaseSkeleton />;
+    }
+
+    if (!data) {
+        return <Alert color="error">Nous n&apos;as vont pas peu récupérais les donnée.</Alert>;
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
        return <Alert color="error">Nous n&apos;avons pas pu récupérer les données.</Alert>;
```